dynamiq/memory/backend/in_memory.py

```diff
@@ -8,6 +11,46 @@ class InMemoryError(Exception):
     pass
 
 
+class BM25:
```

Comment:
Please improve the naming as `BM25` is not clear.

Reply:
It's just the name of algorithm - standart in field, from here
```
https://en.wikipedia.org/wiki/Okapi_BM25
``

Reply:
I mean `BM25DocumentRanker` or smth like this